CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## Unreleased
+
+Remove the `table-layout: fixed` style from the tables in tech' docs content, mainly due to it
```

Comment:
Not blocking by ideally the changelog should say what is fixed for users, not the details of what we've been doing internally.

Reply:
Makes sense. Does [this change](https://github.com/alphagov/tech-docs-gem/pull/154/commits/1f90f78c00bbe7f1426aacc476dcde1325284db3) help?

Reply:
Great, I'd prefer if it didnt require looking in the link to understand the summary but good enough 👍 Thanks Tom

Reply:
That's a good point actually. I've rewritten it in [this commit](https://github.com/alphagov/tech-docs-gem/pull/154/commits/a4a20f19eaf412c8e455768f974eb824f1374d8d).